app/views/box_requests/show.json.jbuilder

```diff
@@ -1 +1 @@
-json.partial! "box_requests/box_request", box_request: @box_request
+json.partial! "box_request_reviews/box_request_review", box_request: @box_request
```

Comment:
Is this a partial that needs to be added? I don't see it already on the develop branch.

Reply:
@gabezurita hey! do you know the answer to @h-m-m 's q above?

Reply:
Hi Mae! So sorry for the slow reply, I've been traveling through Chile and will not be back to the US until August 19. Is my input still needed here? Seem as if you've picked up the ball?

I can go back to working on issues once I'm back to the USA--I definitely still want

Reply:
@gabezurita hey there! NO problem at all! yes, i figured it out and handled this thread. enjoy chile! we'll be here when you get back. :) thank you for everything!